src/app/record/components/funding-stack/funding-stack.component.scss

```diff
@@ -0,0 +1,22 @@
+[body] {
+  display: flex;
+  flex-direction: column;
+}
+
+:host {
+  width: 100%;
+}
+
+:host.display-the-stack {
+  border-style: solid;
+  border-radius: 4px;
+  border-width: 2px;
+
+  app-panel:not(.last) {
+    -webkit-box-shadow: 0px 9px 5px 0px rgba(214, 214, 214, 1);
```

Comment:
I am moving this color into a palette in another PR. Let's merge this one as it is.